packages/strapi-utils/lib/content-types.js

```diff
@@ -173,9 +178,13 @@ const getGlobalId = (model, modelName, prefix) => {
   return model.globalId || _.upperFirst(_.camelCase(globalId));
 };
 
+const isRelationalAttribute = attribute =>
+  _.has(attribute, 'model') || _.has(attribute, 'collection');
+
```

Comment:
Is an attribute either `scalar` or `relational`? If yes, was about using `return !isScalarAttribute()` ?

Reply:
In case we add new types it woudl break. better to have eahc type with their own definition